core/app/models/spree/refund.rb

```diff
@@ -43,12 +38,15 @@ def perform!
       credit_cents = Spree::Money.new(amount.to_f, currency: payment.currency).money.cents
 
       @response = process!(credit_cents)
+      log_entries.create!(details: @response.to_yaml)
```

Comment:
This change is great, and allows another refactor here.

We can change this ugly instance variable to just a local variable in this `perform!` method (the only place it is used)